tests/integration/tox.ini

```diff
@@ -34,7 +34,6 @@ deps =
     -r {toxinidir}/requirements-test.txt
 commands =
     pytest -vv \
-        --maxfail 1 \
         --tb native \
         --log-cli-level DEBUG \
         --log-format "%(asctime)s %(levelname)s %(message)s" \
```

Comment:
ok, so this will direct pytest to just run all the tests.  This will be nice to see all the failures created by this pr rather than just the first one.  It should help us change more of the code on the next commit.